libraries/src/Mail/Mail.php

```diff
@@ -237,9 +237,9 @@ public function setSender($from)
 			else
 			{
 				// If it is neither, we log a message and throw an exception
-				Log::add(\JText::sprintf('JLIB_MAIL_INVALID_EMAIL_SENDER', $from), Log::WARNING, 'jerror');
+				Log::add(\JText::sprintf('JLIB_MAIL_INVALID_EMAIL_SENDER', $from, $from), Log::WARNING, 'jerror');
 
-				throw new \UnexpectedValueException(sprintf('Invalid email Sender: %s, Mail::setSender(%s)', $from));
+				throw new \UnexpectedValueException(sprintf('Invalid email Sender: %s, Mail::setSender(%s)', $from, $from));
```

Comment:
```suggestion
				throw new \UnexpectedValueException(sprintf('Invalid email Sender: %1$s, Mail::setSender(%1$s)', $from));
```

Personally, I think it is better not to pass the same argument twice and use the format string argument numbering feature.

Reply:
If that works it is better of course. I did not know from scratch since I'm not really a PHP programmer and wanted to provide a quick solution. Since it's a bit time until 3.10.1 and we are busy today a bit, I will pick it up tomorrow or in the next days.

Reply:
Thanks for the suggestion. I have changed back to draft status so I have time to do that and nobody tests or merges before I've made the change.